File: linked list/display elements.java
import java.io.*;
class node
{
    int data;
    node next;
    node(int a)
    {
    data=a;
    next=null;
}
}
class linkedlist {
	public static void main (String[] args) {
	    node head=new node(10);
	    node first=new node(20);
	   node second=new node(30);
	    head.next=first;
	    first.next=second;
	    second.next=null;
	    node current=head;
	    while(current!=null)
	    {
	        System.out.println(current.data);
	        current=current.next;
	    }
	    }
	}

File: linked list/display elements.c
#include <stdio.h>
#include <malloc.h>
#include <stdlib.h>
struct node
{
  int data;
  struct node *next;
};
typedef struct node node;
node *head,*one,*two,*three,*current;
int main(void)
{
    head=(node *)malloc(sizeof(node));
    one=(node *)malloc(sizeof(node));
    two=(node *)malloc(sizeof(node));
    three=(node *)malloc(sizeof(node));
    head->data=20;
    head->next=two;
    two->data=60;
    two->next=three;
    three->data=80;
    three->next=NULL;
    current=head;
    while(current!=NULL)
    {
        printf("%d\t",current->data);
        current=current->next;
    }
    }
    
